lib/Echidna/Transaction.hs

```diff
@@ -250,6 +250,13 @@ shrinkTx tx'@(Tx c _ _ _ gp (C _ v) (C _ t, C _ b)) = let
     ]
   in join (uniform possibilities) <*> pure tx'
 
+mutateTx :: (MonadRandom m, Has GenDict x, MonadState x m, MonadThrow m) => Tx -> m Tx
+mutateTx (Tx (SolCall c) a b d w x y) = do f <- weighted [(mutate, 10), (skip, 90)] --FIXME: make a parameter for this?
```

Comment:
what about FIXME?

Reply:
before arriving here, there is already a chance of skipping with `TxIdentity`, what's the purpose of this additional 90% skip?

Reply:
I simplified some mutations and refactored the code.